src/strategies/cyberkongz/index.ts

```diff
@@ -28,15 +43,25 @@ export async function strategy(
 
   const response = await multicall(network, provider, abi, calls, { blockTag });
   const nanaCall = [
-    ['0xe2311ae37502105b442bbef831e9b53c5d2e9b3b', 'totalSupply', []]
+    [bananaContract, 'totalSupply', []]
   ];
-  const nanaSupply = await multicall(network, provider, abi, nanaCall, {
+  let nanaSupply = await multicall(network, provider, abi, nanaCall, {
     blockTag
   });
 
+  response.forEach((value: any, i: number) => {
+    const address = calls[i][2][0];
+    if (Math.floor(i / addresses.length) == 2 && blacklistedAddresses.find((add) => add === address)) {
+      nanaSupply -= value;
+    }    
+  });
+
```

Comment:
Hello @frosti-eth we can't update this as it can affect existing space that are using this strategy